client/crashpad_client.h

```diff
@@ -162,6 +162,18 @@ class CrashpadClient {
   static int ConsecutiveCrashesCount(const base::FilePath& database);
 #endif
 
+#if BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_LINUX) || DOXYGEN
+  //! \brief Enable the use of a GUID override.
+  //!
+  //! This function must be called prior to `StartHandler()`
+  //!
+  //! \param[in] uuid The GUID to use as the GUID.
+  //!
+  //! \return `true` on success. Otherwise `false` with a message logged.
+  bool EnableGuidOverride(const std::string& uuid);
```

Comment:
wouldn't it be better if we rename it to OverrideGuid?

Reply:
I have no particular opinion. I could change the name.

Reply:
I think OverrideGuid would be better - the method overrides the guid so I just feel this name will be better